lib/tasks/grade.rake

```diff
@@ -92,6 +97,42 @@ namespace :grade do
 
 end
 
+def sync_specs_with_source
+  reponame = Dir.pwd.split("/").last
+  full_reponame = "appdev-projects/#{reponame}"
+  spec_folder_contents = Octokit.contents(full_reponame, :path => 'spec')
+  spec_subfolder_name = spec_folder_contents.first[:name]
```

Comment:
My thought process was that the more files that are downloaded, the slower `rails grade` would become. By knowing the subfolder name I can directly download each of the relevant spec files instead of having to replace files that don't impact grade scores. Although, your suggestion in [a later comment](https://github.com/firstdraft/grade_runner/pull/57#discussion_r994095696) might be faster than this loop approach.